ISSUE_TEMPLATE.md

```diff
@@ -1,100 +1,123 @@
 Make sure you have provided the following information:
 
- - [ ] link to your code branch cloned from rhboot/shim-review in the form user/repo@tag
- - [ ] completed README.md file with the necessary information
+ - [ ] Link to your code branch cloned from rhboot/shim-review in the form user/repo@tag
+ - [ ] Completed README.md file with the necessary information
  - [ ] shim.efi to be signed
- - [ ] public portion of your certificate(s) embedded in shim (the file passed to VENDOR_CERT_FILE)
- - [ ] binaries, for which hashes are added do vendor_db ( if you use vendor_db and have hashes allow-listed )
- - [ ] any extra patches to shim via your own git tree or as files
- - [ ] any extra patches to grub via your own git tree or as files
- - [ ] build logs
- - [ ] a Dockerfile to reproduce the build of the provided shim EFI binaries
+ - [ ] Public portion of your certificate(s) embedded in shim (the file passed to VENDOR_CERT_FILE)
+ - [ ] Binaries, for which hashes are added do vendor_db ( if you use vendor_db and have hashes allow-listed )
+ - [ ] Any extra patches to shim via your own git tree or as files
+ - [ ] Any extra patches to grub via your own git tree or as files
+ - [ ] Build logs
+ - [ ] A Dockerfile to reproduce the build of the provided shim EFI binaries
```

Comment:
Maybe drop these capslock changes -- I think anything that shrinks the diff makes a merge more likely

Reply:
Done